src/Data/BoolExpr.hs

```diff
@@ -43,6 +43,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -- Module    : Data.BoolExpr
 -- Copyright : (c) Nicolas Pouillard 2008,2009
 -- License   : BSD3
+-- SPDX-License-Identifier: BSD3
```

Comment:
This should be `BSD-3-Clause`, I don't think `BSD3` is a [valid SPDX license ID](https://spdx.org/licenses/).